File: PYTHON/Questões Uri/1072.py
qtd=int(input())
dentro=0
fora= 0

for x in range(0,qtd,1):
  x=int(input())
  if (x>=10 and x<=20):
    dentro=dentro+1
  else:
    fora=fora+1


print(dentro,"in")
print(fora,"out")

File: ESTUDO DE LINGUAGENS/JAVASCRIPT/Questões uri/1072.js
let qtd=+lines.shift()
let dentro=0
let fora=0
let x

for (let cont =0; cont<qtd; cont++){
    x=+lines.shift()

    if (x>=10 && x<=20){
        dentro=dentro+1
    }else{
        fora=fora+1
    }
}
console.log(`${dentro} in`)
console.log(`${fora} out`)